File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/models/Position.java
package com.eulersbridge.isegoria.network.api.models;

import android.support.annotation.Nullable;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

@Parcel
public class Position {

    @Json(name = "positionId")
    public long id;

    @Json(name = "electionId")
    @Nullable
    public Long electionId;

    public String name;
    public String description;
}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/models/Position.kt
package com.eulersbridge.isegoria.network.api.models

import android.annotation.SuppressLint
import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@SuppressLint("ParcelCreator")
@Parcelize
data class Position (
    @field:Json(name = "positionId") var id: Long = 0,
    @field:Json(name = "electionId") val electionId: Long?,
    val name: String?,
    val description: String?
) : Parcelable
